src/javascript/app/pages/user/account/change_password.js

```diff
@@ -1,45 +1,259 @@
+            { selector: '#new_trading_password', request_field: 'new_password', validations: ['req', 'password', ['not_equal', { to: '#old_trading_password', name1: localize('Current password'), name2: localize('New password') }], 'compare_to_email'] },
+            { request_field: 'trading_platform_password_change', value: 1 },
+        ]);
+        FormManager.handleSubmit({
+            form_selector       : trading_form_id,
+            fnc_response_handler: tradingPwHandler,
+            enable_button       : true,
+        });
+
+        // Handle forgot trading password
+        $(forgot_trading_pw_btn_id).off('click').on('click', () => {
+            onSentEmail('trading_password');
+        });
+    };
+
+    const initSetTradingPassword = () => {
+        $set_trading_password_container.removeClass('invisible');
+
+        // Handle set trading password
+        FormManager.init(trading_form_id, [
+            { selector: '#set_new_trading_password', request_field: 'new_password', validations: ['req', 'password', 'compare_to_email'] },
+            { request_field: 'old_password', value: '' },
+            { request_field: 'trading_platform_password_change', value: 1 },
+        ], null, shouldSetTradingPassword());
+        FormManager.handleSubmit({
+            form_selector       : trading_form_id,
+            fnc_response_handler: setTradingPwHandler,
         });
     };
 
-    const handler = (response) => {
+    const changePasswordHandler = (response) => {
         if ('error' in response) {
-            $(`${form_id}_error`).text(response.error.message).setVisibility(1);
+            const frm_change_binary_password_error_id = '#frm_change_binary_password_error';
+            $(frm_change_binary_password_error_id).text(response.error.message).setVisibility(1);
+            setTimeout(() => {
+                $(frm_change_binary_password_error_id).setVisibility(0);
+            }, 5000);
         } else {
-            $(form_id).setVisibility(0);
-            $('#msg_success').setVisibility(1);
+            $msg_success.text(localize('Your binary password has been changed. Please log in again.'));
+            $msg_success_container.setVisibility(1);
             setTimeout(() => {
                 Client.sendLogoutRequest(true);
             }, 5000);
         }
     };
 
+    const setTradingPwHandler = (response) => {
+        if ('error' in response) {
+            const frm_set_trading_password_error_id = '#frm_set_trading_password_error';
+            $(frm_set_trading_password_error_id).text(response.error.message).setVisibility(1);
+            setTimeout(() => {
+                $(frm_set_trading_password_error_id).setVisibility(0);
+            }, 5000);
+        } else {
+            $msg_success_trading.text(localize('Your trading password has been set. Please use this to log in to MetaTrader 5 and Deriv X.'));
```

Comment:
deriv x is not launched yet. also it's not planning to be launched in binary.

Reply:
Wrong copy, I will fix it.